src/addrman.cpp

```diff
@@ -868,28 +868,28 @@ void AddrManImpl::ResolveCollisions_()
                 AddrInfo& info_old = mapInfo[id_old];
 
                 // Has successfully connected in last X hours
-                if (GetAdjustedTime() - info_old.nLastSuccess < ADDRMAN_REPLACEMENT_HOURS*(60*60)) {
+                if (count_seconds(GetTime<std::chrono::seconds>()) - info_old.nLastSuccess < ADDRMAN_REPLACEMENT_HOURS*(60*60)) {
                     erase_collision = true;
-                } else if (GetAdjustedTime() - info_old.nLastTry < ADDRMAN_REPLACEMENT_HOURS*(60*60)) { // attempted to connect and failed in last X hours
+                } else if (count_seconds(GetTime<std::chrono::seconds>()) - info_old.nLastTry < ADDRMAN_REPLACEMENT_HOURS*(60*60)) { // attempted to connect and failed in last X hours
 
                     // Give address at least 60 seconds to successfully connect
-                    if (GetAdjustedTime() - info_old.nLastTry > 60) {
+                    if (count_seconds(GetTime<std::chrono::seconds>()) - info_old.nLastTry > 60) {
                         LogPrint(BCLog::ADDRMAN, "Replacing %s with %s in tried table\n", info_old.ToString(), info_new.ToString());
 
                         // Replaces an existing address already in the tried table with the new address
-                        Good_(info_new, false, GetAdjustedTime());
+                        Good_(info_new, false, count_seconds(GetTime<std::chrono::seconds>()));
                         erase_collision = true;
                     }
-                } else if (GetAdjustedTime() - info_new.nLastSuccess > ADDRMAN_TEST_WINDOW) {
+                } else if (count_seconds(GetTime<std::chrono::seconds>()) - info_new.nLastSuccess > ADDRMAN_TEST_WINDOW) {
                     // If the collision hasn't resolved in some reasonable amount of time,
                     // just evict the old entry -- we must not be able to
                     // connect to it for some reason.
                     LogPrint(BCLog::ADDRMAN, "Unable to test; replacing %s with %s in tried table anyway\n", info_old.ToString(), info_new.ToString());
-                    Good_(info_new, false, GetAdjustedTime());
+                    Good_(info_new, false, count_seconds(GetTime<std::chrono::seconds>()));
                     erase_collision = true;
                 }
             } else { // Collision is not actually a collision anymore
-                Good_(info_new, false, GetAdjustedTime());
+                Good_(info_new, false, count_seconds(GetTime<std::chrono::seconds>()));
```

Comment:
`count_seconds` should only be used to *serialize* the time value in seconds. For example for RPC, the debug log, or the GUI, ...

It shouldn't be used for normal code. You can simply use the equivalent `GetTime` here. If you want to use std::chrono, you should update all related types, but this should be done in a separate commit. Ideally a separate pull request.

Reply:
Changed to  `GetTime`.